MAUI/DataGrid/Getting-started.md

```diff
@@ -0,0 +1,341 @@
+---
+layout: post
+title: Getting Started with .NET MAUI DataGrid control | Syncfusion
+description: Learn here about getting started with Syncfusion .NET MAUI DataGrid (SfDataGrid) control, its elements and more.
```

Comment:
description: Learn about getting started with Syncfusion .NET MAUI DataGrid (SfDataGrid) control, its elements, and more here.